docs/migration.md

```diff
@@ -1204,7 +1214,131 @@ Docker Logs Source is not supported by the OpenTelemetry Collector.
+
+#### Overall example
+
+Below is an example of an OpenTelemetry configuration for a Docker Stats Source.
+
+```yaml
+receivers:
+  docker_stats:
+    ## Docker daemon's socket address.
+    ## Example for running Docker on MacOS with Colima, default is "unix:///var/run/docker.sock"
+    endpoint: "unix:///Users/<user>/.colima/default/docker.sock"
+    collection_interval: 15s
+
+    ## A list of images for which corresponding containers won't be scraped.
+    ## Strings, regexes and globs are supported, more information in the receiver's readme:
+    ## https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.73.0/receiver/dockerstatsreceiver#configuration
+    excluded_images:
+      ## Regexes must be put between / characters
+      - /.*nginx/
+    timeout: 10s
+    ## Must be 1.22 or above
+    api_version: 1.23
+
+    ## Map Docker container labels to resource attributes
+    container_labels_to_metric_labels:
+      com.docker.compose.container-number: container-number
+    ## Map Docker container environmental variables to resource attributes
+    env_vars_to_metric_labels:
+      BIN_DIR: bin-directory
+
+    ## Enable or disable particular metrics.
+    ## Full list of metrics with their default config is available at https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.73.0/receiver/dockerstatsreceiver/documentation.md
+    metrics:
+      container.cpu.usage.percpu:
+        enabled: true
+      container.network.io.usage.tx_dropped:
+        enabled: false
+```
+
+#### Name
+
+Please refer to [the Name section of Common configuration](#name-1).
+
+#### Description
+
+Please refer to [the Description section of Common configuration](#description-1).
+
+#### URI
+
+To specify URI, use `endpoint` option:
+
+```yaml
+receivers:
+  dockerstats:
+    endpoint: http://example.com
+```
+
+`Cert Path` option is not supported in OpenTelemetry Collector.
+
+#### Container filters
+
+Unfortunately, containers cannot be filtered by their name in OpenTelemetry Collector. The only way to filter containers is to exclude them by their image name through `excluded_images` option:
+
+```yaml
+receivers:
+  dockerstats/1:
+    excluded_images:
+      ## Exclude particular image
+      - docker.io/library/nginx:1.2
+      ## Exclude by regex
+      ## Note: regex must be but between / characters
+      - /other-docker-registry.*/
+      ## Exclude by glob
+      - exclude-*-this
+
+  dockerstats/2:
+    excluded_images:
+      ## Use negation: scrape metrics only from nginx containers
+      - !*nginx*
+      ## Negation for regexes requires using ! before the slash character
+      - !/.*nginx.*/
+```
+
+Valid names are strings, [regexes](https://pkg.go.dev/regexp) and [globs](https://github.com/gobwas/glob).
+Negation can also be used, for example glob `!my*container` will exclude all containers for which the image name doesn't match glob `my*container`.
```

Comment:
Upstream docs describe negations in a weird way, I've opened a pull request there to clarify that: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19423